system/loggerd/deleter.py

```diff
@@ -52,15 +73,19 @@ def deleter_thread(exit_event):
 
     if out_of_percent or out_of_bytes:
       dirs = listdir_by_creation(Paths.log_root())
+      segments, non_segments = separate_segments_and_nonsegments(dirs)
```

Comment:
it's nice that deleter doesn't really know much about "segments". let's keep it that way

Reply:
done